lib/rucio/client/downloadclient.py

```diff
@@ -244,7 +244,7 @@ def download_pfns(self, items, num_threads=2, trace_custom_fields={}, traces_cop
 
         return self._check_output(output_items)
 
-    def download_dids(self, items, num_threads=2, trace_custom_fields={}, traces_copy_out=None):
+    def download_dids(self, items, num_threads=2, trace_custom_fields={}, traces_copy_out=None, test_module=False):
```

Comment:
hmmm, did you discuss this with @bari12 ? having the test_module in the public signature looks odd to me

Reply:
@mlassnig no i had not discussed it, what would you suggest instead for it to be more efficient?